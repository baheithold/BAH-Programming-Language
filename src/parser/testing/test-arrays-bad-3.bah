#This file tests a invalid arrays


[]arr = [];
