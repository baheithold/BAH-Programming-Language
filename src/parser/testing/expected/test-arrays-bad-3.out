ERROR, line 4: expected SEMICOLON, but encountered ID_TYPE
